common/lib/client/realtimechannel.ts

```diff
@@ -105,47 +158,30 @@ var RealtimeChannel = (function() {
 			 * rejecting messages until we have confirmation that the options have changed,
 			 * which would unnecessarily lose message continuity. */
 			this.attachImpl();
-			this._allChannelChanges.once(function(stateChange) {
+			this._allChannelChanges.once(function(this: any, stateChange: ConnectionStateChange) {
```

Comment:
shouldn't this be {event: string} rather than any?

Reply:
Fixed